llm_on_ray/inference/serve.py

```diff
@@ -31,7 +30,7 @@ def get_deployed_models(args):
     2. Use relevant configuration parameters to generate `InferenceConfig` if model_id_or_path is set,
     3. Serve all pre-defined models in inference/models/*.yaml, or part of them if models is set.
```

Comment:
Due to the resource constraint, I don't think we can serve all models by default if none were chosen. better to default to use gpt2 if none where chosen.